auth/client_test.go

```diff
@@ -0,0 +1,36 @@
+package auth_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-notification/auth"
+	"github.com/fabric8-services/fabric8-notification/auth/api"
+	"github.com/goadesign/goa/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	openshiftIOAPI = "http://auth.openshift.io"
+)
+
+func createClient(t *testing.T) *api.Client {
+	c, err := auth.NewCachedClient(openshiftIOAPI)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return c
+}
+
+func TestSpaceCollaborators(t *testing.T) {
+
+	c := createClient(t)
+	id, _ := uuid.FromString("020f756e-b51a-4b43-b113-45cec16b9ce9")
+
+	u, err := auth.GetSpaceCollaborators(context.Background(), c, id)
```

Comment:
yeah this test was there before, just moved it around a bit to a new file.
using the prod space "openshiftio" because that's the only one's that's constant and hence test-able. Ideal situation would be to create a space and validate it :)